packages/mco/components/mco-dashboard/data-policy/cluster-app-card/cluster.tsx

```diff
@@ -354,23 +321,33 @@ export const UtilizationCard: React.FC<UtilizationCardProps> = ({
       <div className="mco-cluster-app__graph--margin-bottom">
         <SnapshotUtilizationCard
           clusters={peerClusters}
-          title={t('Snapshots synced')}
+          title={t('Block volumes snapshots synced')}
           queryType={DRDashboard.RBD_SNAPSHOT_SNAPSHOTS}
-          titleToolTip={t(
-            'The y-axis shows the number of snapshots taken. It represents the rate of difference in snapshot creation count during a failover.'
-          )}
+          titleToolTip={
+            <Trans t={t}>
+              The graph displays the total number of block volumes inbound
+              snapshots, by cluster, from all AppicationSet and Subscription
+              type applications. Applications that use file volumes are excluded
+              in the total snapshot count.
+            </Trans>
```

Comment:
```suggestion
   
            t("
              The graph displays the total number of block volumes inbound
              snapshots, by cluster, from all AppicationSet and Subscription
              type applications. Applications that use file volumes are excluded
              in the total snapshot count.")
```
try t instead of <trans>,  we dont have any tags inside the string. And test once how it looks like in tooltip,